.travis.yml

```diff
@@ -30,6 +30,17 @@ script:
     - make $MAKEJOBS || ( echo "Build failure. Verbose build follows." && make V=1 ; false )
     - make check $MAKEJOBS VERBOSE=1
 
+jobs:
+  include:
+    - stage: "Benchmark: Linux + GCC"
+      os: linux
```

Comment:
We can run them on osx too. But I am not sure how we would use the output except that knowing that they were run without errors.

Reply:
As a far as I know we can setup Travis in the future so that it will post message commit with bencmark results back to a pull request.

Let me add osx a bit later. I'm not at home now